frontend/src/hooks/api/getters/useConstraintsValidation/useConstraintsValidation.ts

```diff
@@ -16,25 +24,29 @@ export const useConstraintsValidation = (
             return;
         }
 
-        const validationRequests = constraints
-            .filter(constraint => {
-                const hasValues =
-                    Array.isArray(constraint.values) &&
-                    constraint.values.length > 0;
-                const hasValue = Boolean(constraint.value);
+        const invalidConstraints = constraints.find(item => !isValid(item));
```

Comment:
check locally invalid constraints that don't require server checks

Reply:
do we highlight what's wrong? it can be confusing when button is disabled and there is no validation error